src/Mvc/Mvc.Core/src/ModelBinding/Binders/SimpleTypeModelBinder.cs

```diff
@@ -46,6 +46,8 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
                 throw new ArgumentNullException(nameof(bindingContext));
             }
 
+            _logger.AttemptingToBindModel(bindingContext);
```

Comment:
Do we need move the logging above the work in other binders?

Reply:
Of the ones that relied on the value provider, Simple type was the only outlier.